src/core/dev/cq_mgr.h

```diff
@@ -137,7 +137,7 @@ class cq_mgr {
      */
     virtual int poll_and_process_element_rx(uint64_t *p_cq_poll_sn, void *pv_fd_ready_array = NULL);
     virtual int poll_and_process_element_tx(uint64_t *p_cq_poll_sn);
-    virtual mem_buf_desc_t *poll_and_process_socketxtreme();
+    virtual mem_buf_desc_t *poll_and_process_socketxtreme() { return nullptr; };
```

Comment:
You removed `cq_logerr("SocketXtreme mode is supported by mlx5 cq manager only");`
Do we have any other notification not to use socketxtreme on pure verbs?

Reply:
We will remove pure verbs completely soon. And it did nothing anyway.